db/migrate/20150314144907_add_seed_recipes_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddSeedRecipesToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :seed_recipes, :boolean, :default => false
```

Comment:
Not that it really matters much, but for consistency, maybe we should use the 1.9 hash syntax here? That is, replace `:default =>  true` with `default: true`.